gen/gen_component_impl.py

```diff
@@ -135,7 +136,7 @@ def main():
 
     for fl in files:
         fields = []
-        component_name = fl.split('/')[-1]
+        component_name = fl.split(os.sep)[-1]
```

Comment:
Is glob.glob actually giving you back paths with '\' rather than '/' on windows?